<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>МагизинМузыкальных инструментов</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/newPrefix/res/icon.ico</normaloff>:/newPrefix/res/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Goods</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1" colspan="2">
            <widget class="QTableWidget" name="browser">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>Brand</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Price</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Model</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type Name</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_remove">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btn_save">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Сохранить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Brand</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_brand"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Price</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="input_price">
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Model</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_model"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Type Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="input_typename">
             <item>
              <property name="text">
               <string>Струнные</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Духовые</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ударные</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Медиаторы</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Комбоусилители</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Клавишные</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Добавление товара</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Brand</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Price</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="input_price_2">
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Type Name</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QComboBox" name="input_typename_2">
          <item>
           <property name="text">
            <string>Струнные</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Духовые</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ударные</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Медиаторы</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Комбоусилители</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Клавишные</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="input_brand_2"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="input_model_2"/>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="btn_create">
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>WarehousesToGoods</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0" colspan="2">
            <widget class="QTableWidget" name="browser_2">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>Warehouse</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Product</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Number</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_remove_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_save_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Сохранить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Warehouse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="input_warehouse"/>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Product</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="input_product"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="input_number">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Добавление в таблицу WarehouseToGoods</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="5" column="0">
            <widget class="QSpinBox" name="input_number_2">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QPushButton" name="btn_create_2">
             <property name="text">
              <string>Cоздать</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="input_warehouse_2"/>
           </item>
           <item row="3" column="0">
            <widget class="QComboBox" name="input_product_2"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Warehouse</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Number</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Product</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
